tst.so/src/tst_worker.cpp

```diff
@@ -319,7 +299,7 @@ bool TSTWorker::do_thermal_stress(void) {
         std::to_string(gpu_id) + " " + "max. junction temperature :" + " " + std::to_string(max_junction_temperature);
     rvs::lp::Log(msg, rvs::loginfo);
 
-    result = true;
+    //result = true;
```

Comment:
Any reason this is commented ?

Reply:
line is useless as we follow this with if else clause that appropriately updates result variable and result is anyway initialized to true

Reply:
Ok. We can remove it then.